src/editors/data/redux/thunkActions/requests.test.js

```diff
@@ -101,77 +113,64 @@ describe('requests thunkActions module', () => {
       });
     });
   };
-
   describe('network request actions', () => {
-    const submissionUUID = 'test-submission-id';
-    const locationId = 'test-location-id';
+    const sometestStateGetter = () => ({
+      some: 'data',
+    });
+
     beforeEach(() => {
       requests.networkRequest = jest.fn(args => ({ networkRequest: args }));
     });
-    describe('initializeApp', () => {
-      testNetworkRequestAction({
-        action: requests.initializeApp,
-        args: { locationId },
-        expectedString: 'with initialize key, initializeApp promise',
-        expectedData: {
-          requestKey: RequestKeys.initialize,
-          promise: api.initializeApp(locationId),
-        },
-      });
-    });
-    describe('fetchSubmissionStatus', () => {
-      testNetworkRequestAction({
-        action: requests.fetchSubmissionStatus,
-        args: { submissionUUID },
-        expectedString: 'with fetchSubmissionStatus promise',
-        expectedData: {
-          requestKey: RequestKeys.fetchSubmissionStatus,
-          promise: api.fetchSubmissionStatus(submissionUUID),
-        },
-      });
-    });
-    describe('fetchSubmission', () => {
-      testNetworkRequestAction({
-        action: requests.fetchSubmission,
-        args: { submissionUUID },
-        expectedString: 'with fetchSubmission promise',
-        expectedData: {
-          requestKey: RequestKeys.fetchSubmission,
-          promise: api.fetchSubmission(submissionUUID),
-        },
-      });
-    });
-    describe('setLock: true', () => {
+    describe('fetchBlock', () => {
       testNetworkRequestAction({
-        action: requests.setLock,
-        args: { submissionUUID, value: true },
-        expectedString: 'with setLock promise',
+        action: requests.fetchBlock,
+        args: { some: 'data' },
+        testState: sometestStateGetter,
+        expectedString: 'with fetchBlock promise',
         expectedData: {
-          requestKey: RequestKeys.setLock,
-          promise: api.lockSubmission(submissionUUID),
+          ...sometestStateGetter(),
+          requestKey: RequestKeys.fetchBlock,
+          promise: api.fetchBlockById({
+            studioEndpointUrl: selectors.app.studioEndpointUrl(sometestStateGetter()),
+            blockId: selectors.app.blockId(sometestStateGetter()),
+          }),
```

Comment:
The args need to be different, as they are specifically fetch parameters.
With the above changes, this becomes:
```
      const fetchParams = { fetchParam1: 'param1', fetchParam2: 'param2' };
      testNetworkRequestAction({
        action: requests.fetchBlock,
        args: fetchParams,
        expectedString: 'with fetchBlock promise',
        expectedData: {
          ...fetchParams,
          requestKey: RequestKeys.fetchBlock,
          promise: api.fetchBlockById({
            studioEndpointUrl: selectors.app.studioEndpointUrl(testState),
            blockId: selectors.app.blockId(testState),
          }),
        },
      });
```